### PR TITLE
Add setting to enable/disable auto-updating on startup

### PR DESCRIPTION
**Pull request by weedgrease**
cc: @jminardi

### CODE REVIEW

File: octoprint_firmwareupdate/__init__.py [hunk 1]
```diff
@@ -52,28 +52,45 @@ def __init__(self):
...
+            eventManager().fire(Events.SETTINGS_UPDATED)
         else:
             self._logger.info("Unknown command: " + command)
 
     def on_api_get(self, request):
         return flask.jsonify(isUpdating=self.isUpdating)
 
     def on_after_startup(self):
-        self._start_update(True)
+        if self._settings.get_boolean(["auto_update"]):
```
Comment (jencheng) [inline octoprint_firmwareupdate/__init__.py:90]: what happens if "auto_update" doesn't exist in settings?

Comment (weedgrease) [inline octoprint_firmwareupdate/__init__.py:90]: `get_settings_defaults()` allows us to inject default settings. if there's nothing in the settings file, `get_boolean` will return the default

Comment (jencheng) [inline octoprint_firmwareupdate/__init__.py:90]: 🆗

**Comment by jencheng** [comment]
👍

**Pull request merged**